main.go

```diff
@@ -3,30 +3,80 @@ package main
 import (
 	"log"
 	"net"
+	"strings"
 
 	"git.icyphox.sh/taigobot/db"
 	"git.icyphox.sh/taigobot/plugins"
 	"github.com/dgraph-io/badger/v3"
 	"gopkg.in/irc.v3"
 )
 
+// (nojusr): A simple check func to find out if an irc message
+// is supposed to be a CTCP message
+func isCTCPmessage(m *irc.Message) bool {
```

Comment:
Why not a regular message handler that just reacts to prefixes with a 0x01 byte?